junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/DynamicTestTestDescriptor.java

```diff
@@ -74,4 +73,22 @@ public void after(JupiterEngineExecutionContext context) throws Exception {
 		this.dynamicTest = null;
 	}
 
+	private static class DynamicTestInvocation<T> implements InvocationInterceptor.Invocation<T> {
+
+		private static final Logger LOG = LoggerFactory.getLogger(DynamicTestInvocation.class);
+
+		private DynamicTest dynamicTest;
+
+		DynamicTestInvocation(DynamicTest dynamicTest) {
+			this.dynamicTest = dynamicTest;
+		}
+
+		@Override
+		public T proceed() throws Throwable {
+			dynamicTest.getExecutable().execute();
+			return null;
+		}
+
+	}
+
```

Comment:
:x: The change to this file can now be reverted.

Reply:
Done, I also revert the changes to DynamicTestTestDescriptor